scripts/install_KLU_Sundials.py

```diff
@@ -268,11 +268,11 @@ def parallel_download(urls, download_dir):
 try:
     subprocess.run(["make", "--version"])
 except OSError:
-    raise RuntimeError("Make must be installed.")
+    raise RuntimeError("Make must be installed.") from None
```

Comment:
These should not be `from None` as that defeats the point of the check. IIRC, the best practice should look like this -
```py
try:
    ...
except ErrorType as e:
    raise NewErrorType(...) from e
```
This will preserve the stack trace, making it easier to debug for both the users and pybamm developers.

Reply:
Right, my only concern was not to make the error message redundant, but it makes sense. I have fixed it now in 03b3a65f72c0f0a69ce6baa5c1f36ac1f3a75367